### DISCUSSION TITLE
Vitest throws error with Next.js image component with static import

### DISCUSSION THREAD

**Discussion by lucas-santosP** [vitest-dev/vitest#3044] [Q&A]
### Describe the bug

it seems that the Vitest running environment does not recognize the static import in Next.js `Image` component, so when I try to run the test it throws an error saying that width and height are required:
```
FAIL  __tests__/Home.test.tsx > home
Error: Image with src "/public/vercel.svg" is missing required "height" property.

FAIL  __tests__/Home.test.tsx > home
Error: Image with src "/public/vercel.svg" is missing required "width" property.
```

But in Next.js width and height are not required when using static import.

### Reproduction

https://github.com/lucas-santosP/next-image-static-import-with-vitest

After following the Vitest setup in the example [with-vitest](https://github.com/vercel/next.js/tree/canary/examples/with-vitest), to reproduce the error you simply need to add an `Image` component with static import and remove the `width` and `height` props. 

For example:
```tsx
// On top of the file:
import vercelLogo from "../public/vercel.svg";

// [...] In the component
<Image src={vercelLogo} alt="Vercel Logo" />
```

And then run the tests
```
yarn vitest
```

### System Info

```shell
System:
    OS: Windows 10 10.0.22621
    CPU: (12) x64 AMD Ryzen 5 3600 6-Core Processor
    Memory: 4.79 GB / 15.95 GB
  Binaries:
    Node: 18.13.0 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.22.19 - C:\Program Files (x86)\Yarn\bin\yarn.CMD
    npm: 8.19.3 - C:\Program Files\nodejs\npm.CMD
  Browsers:
    Edge: Spartan (44.22621.1413.0), Chromium (111.0.1661.44)
    Internet Explorer: 11.0.22621.1
  npmPackages:
    @vitejs/plugin-react: ^3.0.1 => 3.0.1
    vite: ^4.0.0 => 4.0.4
    vitest: ^0.28.3 => 0.28.3
```

### Used Package Manager

yarn

### Validations

- [X] Follow our [Code of Conduct](https://github.com/vitest-dev/vitest/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guidelines](https://github.com/vitest-dev/vitest/blob/main/CONTRIBUTING.md).
- [X] Read the [docs](https://vitest.dev/guide/).
- [X] Check that there isn't [already an issue](https://github.com/vitest-dev/vitest/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/vitest-dev/vitest/discussions) or join our [Discord Chat Server](https://chat.vitest.dev).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by sheremet-va** [comment]
Vitest does what you told it to do 🤷🏻 I didn't even know Vitest supports Next.js. I would recommend opening an issue in Nex.js repo.

**Comment by lucas-santosP** [comment]
I first tried to open this issue in their repo but with no response until now. So I decided to try here to see if anyone has a possible solution or workaround to that.
And besides that, Vitest runs fine in Next.js since most of the code they are just pure React components.

**Comment by sheremet-va** [comment]
> I first tried to open this issue in their repo but with no response until now. So I decided to try here to see if anyone has a possible solution or workaround to that.

Then it's better to open a discussion or ask people in Discord (https://chat.vitest.dev/), issues are for maintainers to fix.
